draft-ietf-tsvwg-dtls-over-sctp-bis.md

```diff
@@ -481,6 +486,12 @@ connection can be terminated and the associated keying material discarded.
    processing. Making it the ULP responsible for handling any resource
    contention related to large user messages.
 
+   It is mandated that DTLS supports the maximum record size of 2<sup>14</sup> bytes.
```

Comment:
Martin wrote:

"you might find that things fail if you **handshake** with 214 bytes in a record."

Should the requirement be general or should it only be for application data?

Reply:
It was intended for application data, not to apply to other DTLS messages like handshake etc.